tutorials/end_to_end/tutorial02_excitatory_inhibitory_network.ipynb

```diff
@@ -0,0 +1,945 @@
t2mOda/y8Wme1dvXT0SLJTbXzPxkffWY5eehJP2HuP1Sc1IJkOYNKIDk0Z0YPLIDvzMjQ8OKKmm2ZWZiJZsBst+9DbjurhnsqdQ5El6DUCcSXpz4yzTEUKcIoTYz/DvFgBrXakE3L/rDFWshqOnk+zmLlvtftaXw1avEGKbEGKH+/l2AC1EZB9bKgNpWJYM5ODDPZfNYJ/xQwLL7Z7E4BrTMG4uEzQQbKirM9qkKiA8dFEcCYKUp5om6YVk4FQQuPjEPXHFeYf6lm5zPcjSQG6xhHkznfcobWgp8Uc8g6w0Ka284Pa23SWVepQq9KdEjhOuyTJ2YVwaxyPd2V2K0WySWJx50ESMGtjqLgt3lUeFaguD4Exo3HPMYOP6gIEc8CD7R2/M8bn93+shsZCk7a+Z+MjLncsQhnW0YPPO+AZyM80/Yw1yPEyX1CqliWn09haKPEmvAQjTILe72rVRRDRc0bNNQUnSkJZbAcioFOcDuMVQ5i4Ap7r7Hg7gVAB3uRKKbUR0hBu94sPK9sZ6iWicEuliNpzjrmp2qdYQD7KzPrjCZhBNHuFEUThM8TybjD1/HOQIA1mNGUtBjUCYBznOPBP59utNMFTqD/NI6HpO/TjeeeB4EMEbFrQNv6fxRHq66YQPMYJ5kl5sDbJ1cma49CAucT3IktLkQ38ZeTxx6tvR7ZdYeJP0lG2lFt2fSS9IGolFaVn4tnrdenE9ioXZq+PfqB4Siyr314yCvKdy2QyGDWj1Ur7HoZmcgqxBjkc1Lmk3T9JrCMKedJ+Eo2fb2/0r/90C4Ddl7ncOgLcQ0WIAp7jfQUSziOgqABBCbALwfQBPu/++5y4DgIsAXAVgCYClAO4IqxfAewC87GqQfwXgXFHluy9Mg+ysD55627N8/92G4p7PH4f/p2Rki/IgJ5nwl8kEy9uyxznbRhssujdW3SJskl5we//yPccMxvIfn4GpowYC0CcIqscf3UbdMPWM+oRKQdP5qwT+I4u/A1MUi8ht1CLeeTATx0DuVLL8ZTOlulRvtnwJi7pv40xsEkIYz1HSZCw2j3JYohC9zXWKVFDN/ppRkJc7myEM72jBju58okgWzQJrkCvHA4ucQfLYEgttkt7idTvw/iufqHzDmFCsanIhxC8B/JKIPiuE+HUldyqE2AjgZMPyZwB8XPn+JwB/spTbL0G9v0GNHxLSU2Z7PJsyKoU9yqeP9Q8RmzzI08cO8j5He5BLmCQWehIIlThD3vokPbX+sMkHsliU3EEut03SUxd/950z8ezKzZFtViUTcZA63wyZzde4cZBtF96XWjyBAe7MEy3tO+4kvSjk8bTFqE99wVKjfKjGetHzIJOvrE5cj75aTo8+YSNKg1waAZEGcrQGOSqxTDWoZn/NaCgXXKZ937qrF6MHt0Vv2kQSizgv1v2ZDxw+GY8v24jpYwYZ17fmMt6L0wV/fhrXf/JIPBfjOQS4k/Ra/X3N48s2JkprzpRPnFfEIhENk1/c4buLqtekvkHYJD11vUqSG980SW/kwFIHHZEXIjChSd9zmAc5lgZZn6SnbBIaxSJygb8N2RjG2vlHTcGv3n9wZLmSLCShB5nM1/nQ3YfH2t62N18Ui0Qt8lMpiUWS+n76ntJkn6ySarolyoNsqCvusavXTb6AJomDDITFQY6335suOgoHuZMP6wT311XmNx84BIdPHYFBbTkM63B09Fs602dNa1RyUQ+Rfs47DpyAFXPOwJgh7cb1//zEEZjmjnQCwIV/fQY3z3sjVt09BXOYN1Zd1JY4v4BPCCG2yC9uTOJPVK1FfQRPg5xgolMSu6xNkVh878yZ+Mm7DzB60Kz4yvr3fdCkYTht5jjrpnH6TS9kmiEiQpLZuVExo23D82k6khnjBuP0mePws3MOjC4M1eMczA738FdOxAdmJw8TqBI16dC6nfY9lgfZuCz+vaszaUQH9nQ9K5mM6kEubetN5DRcw/0nDsWUkR3usnjHrpZKk2oasEex8CQWxjjIJepsHAPcX1ed4/cajes+eSSyGcJId6Lpxp19z0Am97fBNlk6Dt19OO770gn4/lnOYHeSzHpOmLfgmefIFrUljoGcJeUJRURZAK3Va1LfoOTBMq83DV8l0ZmqEosPHzkF7z1sks+QsHm8Dp/qTPR7+wETvGWtOX9s2psvPtrzjJiI40H+4ql7YXB7DjPGOdIQtW0fODy+4Wg7Ds+DbPE+pulIWrIZ/OG8Q7HvhGCEERO+MG8ak0Z0xDfsrC8B6fzGROR7QYgTxcLnQbacuiST9ICScemLg2ycpKdeQ+fzpBEDPMM5vge59DmnaYdt6F4yWxSLDx+5Ow6ZPAzvmz0JD3/lRNx40VHG/TbA8Cf31zVEyirWb48fC7lZKCd6DFPivCN2x1v3szucTLy4aisWvbk9sJzN49oSnlrM4U4A1xHRFe73T6I0KY6xEOVlM2WAS9IfmSbpZWI8qPccMxgr5pyBp5Zvwq/mLgYA/OMTR8TfMeJFFTh2+mi89J3TSu1x/x4xbYTPOLeh6ntNSLvGppNL0pGk7nSUiYTlPEtsm/oMr4R1Jo2DbIoyUk4UC6BkXGYzaqrpUqW6Tt3ZafBjmhcNL5NeZBvNbfa+u3WOGdKOGy862ls+yY0s4+yjoQwJ7q9ryKhBfddATh1mkgkw1iLDsHHvwrXG5exBri1xnnRfBXAfgE+5/14CMKCajeoLSK+dNCZP1SQLJq9ekvBiJgPZ70EOr0zua4/RA7HX2MGJDLw0MX9N2QDN5Zy/nnzBVg7S+Kq/Ts40ybFS9UoSTdKDnmo6emP13vE0yJaycTzSQCkcoHocfomFX6eu7pNQMqrTnFq5TdR1sUkqbN/D9tUgcH9dQ4YOaEFLlrDBzab31ydW4gvXz6tvoyqE7FrZJiufpAayDb4WtSXySSeEKAJ4EsAKALMBnAQgfvL5foqUWLRkM3jiaycHdK1GiUWiSXrhBnaSTHpO+QQa1xQWQckbGFXOX
```

Comment:
You could add the nice "learn more" section as in the other tutorials.

Reply:
Good point, addressed in 5ab2673.